queue/pom.xml

```diff
@@ -174,4 +174,36 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <shadedArtifactAttached>true</shadedArtifactAttached>
+                            <shadedClassifierName>jdbi-shaded</shadedClassifierName>
+                            <relocations>
+                                <relocation>
+                                    <pattern>org.skife.jdbi</pattern>
+                                    <shadedPattern>org.killbill.queue.org.skife.jdbi</shadedPattern>
```

Comment:
That does not require changing any kb code (e.g `import`) to use the new path?

Reply:
Nope, it creates a new shaded artifact -- it doesn't replace the main one.